htdocs/core/triggers/interface_20_all_Logevents.class.php

```diff
@@ -77,54 +77,58 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
 
 		$date = dol_now();
 
-		// Actions
+		/* Actions */
+
+		$langs->load("users");
```

Comment:
We don't want to load the file if we don't need it. So if action is not a USER_... action, we should not load it.

Reply:
Yes -  I agree,
but before the file was loaded for every action and it seems that it is always needed - if this module is active.
??